Grammar fixes (#1966)
hacks-guide/Guide_3DS: _pages/en_US/installing-boot9strap-(fredtool).txt

1. Select "Verify files"
1. Select "Build and Download"
1. Wait for the process to complete

> 1. When the process has been completed, download your modified DSiWare archive from the site

Reviewer: ```suggestion
1. When the process is complete, download your modified DSiWare archive from the site
```

Reviewer: This was not added.
